sand/extension_interface/search.py

```diff
@@ -1,23 +1,29 @@
 from abc import ABC, abstractmethod
 from typing import Dict
+from sand.models.search import SearchPayload
```

Comment:
let's move the search payload to this interface. storing in models.search is too far.

Reply:
After discussing, we have decided to keep all dataclasses under models module.